src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/wiremock/HmppsAuthMockServer.kt

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.wiremock
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock
+import com.github.tomakehurst.wiremock.http.HttpHeader
+import com.github.tomakehurst.wiremock.http.HttpHeaders
+import org.junit.jupiter.api.extension.AfterAllCallback
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.BeforeEachCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+
+class HmppsAuthApiExtension : BeforeAllCallback, AfterAllCallback, BeforeEachCallback {
```

Comment:
I am a bit confused by all this auth stuff - we don't need to act as an HMPPS auth client. But maybe this is to accept incoming calls?

Reply:
We need it anyway. These extensions are like cross-cutting concerns, which apply set actions to all tests without messing up the test code.